modules/basegl/src/viewmanager.cpp

```diff
@@ -104,7 +102,6 @@ bool ViewManager::propagateMouseEvent(MouseEvent* me, Propagator propagator) {
         newEvent.setPosNormalized(scale * (newEvent.posNormalized() - offset));
         propagator(&newEvent, selectedView_.second);
         if (newEvent.hasBeenUsed()) me->markAsUsed();
-        for (auto p : newEvent.getVisitedProcessors()) me->markAsVisited(p);
```

Comment:
But If it is not a picking event but a regular mouseEvent, that will also use this function, should that not copy the visited processors in that case?

Reply:
Good catch! I committed a fix for it